dbsc/workflow/rules/mrtpipelines.smk

```diff
@@ -1,571 +1,600 @@
 from os.path import join
 
-include: 'zona_bb_subcortex.smk'
-include: 'freesurfer.smk'
 
-# DWI Processing
-rule nii_to_mif:
+include: "zona_bb_subcortex.smk"
+include: "freesurfer.smk"
+
+
+# Directories
+mrtrix_dir = join(config["derivatives"], "mrtrix")
```

Comment:
As far as I can tell `config["derivatives"] == False` by default. Maybe `join(config["out_dir"], derivatives, "mrtrix")` or something like that?